client/signup/steps/site-or-domain/index.jsx

```diff
@@ -29,12 +29,13 @@ import { getTld } from 'lib/domains';
 class SiteOrDomain extends Component {
 	getDomainProductSlug( domain ) {
```

Comment:
Would be a good idea to extract this to a helper, as we'll need it soon...ish. :)